eth/handler_eth_test.go

```diff
@@ -167,7 +164,7 @@ func testForkIDSplit(t *testing.T, protocol uint) {
 			t.Fatalf("frontier nofork <-> profork handler timeout")
 		}
 	}
-	// Progress into Homestead. Fork's match, so we don't care what the future holds
+	// Progress into _. Fork's match, so we don't care what the future holds
```

Comment:
I think the madlibs should have hints for what type of word goes here. Like `<noun>`? :wink: